test/unit/safety_check.c

```diff
@@ -13,14 +13,21 @@ void fake_abort(const char *message) {
 	fake_abort_called = true;
 }
 
+static void
+buffer_overflow_write(char *ptr, size_t size) {
+	/* Avoid overflow warnings. */
+	volatile size_t idx = 128;
```

Comment:
Why is the size argument needed if it's never used?

Reply:
Good catch. Just fixed.